packages/modelserver-client/src/model-server-client.ts

```diff
@@ -216,7 +216,7 @@ export class ModelServerClient implements ModelServerClientApiV1 {
         if (!options.format) {
             options.format = this.defaultFormat;
         }
-        Object.entries(paramOptions).forEach(entry => queryParams.append(entry[0], entry[1]));
+        Object.entries(paramOptions).forEach(entry => queryParams.append(entry[0], entry[1].toString()));
```

Comment:
why the need of the `toString()` call now?

Reply:
This is due to the typescript version update: `entry[1]` is of type 'string | number | boolean' is not assignable to parameter of type 'string'. Hence the appendage to the `URLSearchParams` (string records) requires a conversion to string.